pkg/app/rds_postgres.go

```diff
@@ -141,6 +149,39 @@ func (pdb *RDSPostgresDB) Install(ctx context.Context, ns string) error {
 	if err := kube.WaitOnDeploymentReady(ctx, pdb.cli, pdb.namespace, pdb.bastionDebugWorkloadName); err != nil {
 		return errors.Wrapf(err, "Failed while waiting for deployment %s to be ready, app: %s", pdb.bastionDebugWorkloadName, pdb.name)
 	}
+
+	pdb.vpcID = os.Getenv("VPC_ID")
+
+	// VPCId is not provided, use Default VPC
+	if pdb.vpcID == "" {
+		defaultVpc, err := ec2Cli.DescribeDefaultVpc(ctx)
+		if err != nil {
+			return err
+		}
+		if len(defaultVpc.Vpcs) == 0 {
+			return fmt.Errorf("No default VPC found")
+		}
+		pdb.vpcID = *defaultVpc.Vpcs[0].VpcId
+	}
+
+	// describe subnets in the VPC
+	resp, err := ec2Cli.DescribeSubnets(ctx, pdb.vpcID)
+	if err != nil {
+		return errors.Wrapf(err, "Failed to describe subnets")
+	}
+
+	// Extract subnet IDs from the response
+	var subnetIDs []string
+	for _, subnet := range resp.Subnets {
+		subnetIDs = append(subnetIDs, *subnet.SubnetId)
+	}
+	// create a subnetgroup with subnets in the VPC
+	subnetGroup, err := rdsCli.CreateDBSubnetGroup(ctx, fmt.Sprintf("%s-subnetgroup", pdb.name), "kanister-test-subnet-group", subnetIDs)
+	if err != nil {
+		return errors.Wrapf(err, "Failed to create subnet group")
+	}
+	pdb.dbSubnetGroup = *subnetGroup.DBSubnetGroup.DBSubnetGroupName
+
```

Comment:
Same comment has above, and is there possibility to re-use these between both the apps?

Reply:
CreateDBSubnetGroup should be there in install method, but I have tried refactor the other things, PTAL.